canal/rows.go

```diff
@@ -26,15 +28,18 @@ type RowsEvent struct {
 	Rows [][]interface{}
 	// Header can be used to inspect the event
 	Header *replication.EventHeader
+	// NextPos including binlog name
```

Comment:
Please add comment that when the `RowsEvent` is generated from dump stage, this field is nil.

nit: because your requirement only needs binlog name, how about only store binlog name here? we can discuss it before you accept my comment